xml/admin_saltcluster.xml

```diff
@@ -1376,6 +1376,231 @@ OSD_ID  HOST         STATE                    PG_COUNT  REPLACE  FORCE  STARTED_
+    deployments.
+   </para>
+   <tip>
+    <title>Placement and port usage</title>
+    <para>
+     Deploy the registry on a machine accessible by all nodes in the cluster.
+     We recommend the &adm;. By default, the registry listens on port 5000.
+    </para>
+    <para>
+     On the registry node, use the following command to ensure that the port is
+     free:
+    </para>
+<screen>ss -tulpn | grep :5000</screen>
+    <para>
+     If other processes (such as <literal>iscsi-tcmu</literal>) are already
+     listening on port 5000, determine another free port which can be used to
+     map to port 5000 in the registry container.
+    </para>
+   </tip>
+   <procedure>
+    <step>
+     <para>
+      Verify that the <package>Containers Module</package> extension is
+      enabled:
+     </para>
+<screen>
+&prompt.user;SUSEConnect --list-extensions | grep -A2 "Containers Module"
+Containers Module 15 SP2 x86_64 (Activated)
+</screen>
+    </step>
+    <step>
+     <para>
+      Verify that the following packages are installed:
+      <package>apache2-utils</package> (if enabling a secure registry),
+      <package>cni</package>, <package>cni-plugins</package>,
+      <package>podman</package>, <package>podman-cni-config</package>, and
+      skopeo.
+     </para>
+    </step>
+    <step>
+     <para>
+      Gather the following information:
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+        Fully qualified domain name of the registry host
+        (<option>REG_HOST_FQDN</option>).
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        An available port number used to map to the registry container port of
+        5000 (<option>REG_HOST_PORT</option>).
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        Whether the registry will be secure or insecure
+        (<option>insecure=[true|false]</option>).
+       </para>
+      </listitem>
+     </itemizedlist>
+    </step>
+    <step>
+     <para>
+      Configure the registry by using &cephsalt;:
+     </para>
+<screen>
+&prompt.cephuser;ceph-salt config containers/registries_conf enable
+&prompt.cephuser;ceph-salt config containers/registries_conf/registries \
+ add prefix=<option>REG_HOST_FQDN</option> insecure=[true|false] \
+ location=<option>REG_HOST_PORT</option>:5000
+&prompt.cephuser;ceph-salt apply --non-interactive
+&prompt.cephuser;salt '*' cmd.shell "systemctl restart podman"
+</screen>
+    </step>
+    <step>
+     <para>
+      To start an insecure local registry, create the necessary directory (for
+      example, <filename>/var/lib/registry</filename> and start the registry
+      with the <command>podman</command> command:
+     </para>
+<screen>
+&prompt.root;mkdir -p /var/lib/registry
+&prompt.root;podman run --privileged -d --name registry \
+ -p <option>REG_HOST_PORT</option>:5000 -v /var/lib/registry:/var/lib/registry \
+ --restart=always registry:2
```

Comment:
Would be great, if we could move this piece upstream. There are plenty of folks that would benefit from this. 

See https://tracker.ceph.com/issues/47286

Reply:
To have the registry start after a reboot, we need to make a systemd unit file for it, and enable that service:

```
# podman generate systemd --files --name registry
# mv container-registry.service /etc/systemd/system/
# systemctl enable container-registry.service
```